car_wash/settings.py

```diff
@@ -23,28 +23,33 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-x^av-oc%u-0$fu)c+m4=#yo-m#wcrb$uj#)+n#746job@=yr&s"
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
```

Comment:
add these keys to the .env_example as well.

Reply:
Will push the changes